pep-0008.txt

```diff
@@ -121,6 +121,51 @@ Optional::
       var_one, var_two,
       var_three, var_four)
 
+For dictionaries there is an extra rule. When placing a dictionary key on a new
+line an additional hanging indent should be added. This is so it is easy to
+differentiate between keys and values.
+
+Yes::
+
+    # Keys aligned with opening delimiter, values have an extra indent.
+    mydict = {'firstkey':
+                  'a very very very very very long value',
+              'secondkey': 'a short value',
+              'thirdkey': 'a very very very'
+                  ' long value that continues on the next line'
+                  ' and even on the line after that.',
+    }
+
+    # Keys with one and values with two levels of hanging indents.
+    mydict = {
+        'firstkey':
+            'a very very very very very long value',
+        'secondkey': 'a short value',
+        'thirdkey': 'a very very very'
```

Comment:
I like the option is not done by pycharm the least as well, but I didn't want to make the change to opinionated.